pom.xml

```diff
@@ -278,6 +278,12 @@
       <version>${junit.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.itsallcode</groupId>
+      <artifactId>junit5-system-extensions</artifactId>
```

Comment:
> This is replacement for SystemRules 

if it is replacement .... old library should be removed.
please clarify your intend.

Reply:
Yes, `system-rules` dependency will be removed once all tests migrates to junit5

Reply:
please put it somewhere as TODO, it is so easy to forget.

Reply:
> please put it somewhere as TODO, it is so easy to forget.

Added to the issue checklist: https://github.com/checkstyle/checkstyle/issues/6916#issuecomment-514506599